raiden/transfer/log.py

```diff
@@ -192,6 +224,13 @@ def get_all_state_changes(self):
             for res in self.storage.get_all_state_changes()
         ]
 
+    def get_all_state_events(self):
```

Comment:
yes that is true. I only made it for the tests at the moment. We should not use it in production code.

Maybe I should move it inside the tests directory only?

On the other hand, I have not thought of the proper interface required to query events. Planning to handle it more correctly while working on the [query raiden internal events from the API](https://github.com/raiden-network/raiden/issues/466) issue.